ui/forms.py

```diff
@@ -36,14 +36,10 @@ def __init__(self, *args, **kwargs):
     def clean_course_file(self):
         """Only certain extensions are allowed."""
         upload = self.cleaned_data["course_file"]
-        log.debug("checking filename %s", upload.name)
         for ext in (".zip", ".tar.gz", ".tgz"):
```

Comment:
Just to add to that as an issue, I'd also like to add libmagic checks for the file type.  Those two would go well together when you write the issue